dbld/images/hooks/build

```diff
@@ -10,4 +10,4 @@ DBLD_DIR="./${BUILD_PATH}"
 
 ARG_IMAGE_PLATFORM=$(basename $DOCKERFILE_PATH .dockerfile)
 ${DBLD_DIR}/prepare-image-build $ARG_IMAGE_PLATFORM
-docker build --build-arg=COMMIT=$(git rev-parse --short HEAD) --build-arg=ARG_IMAGE_PLATFORM=${ARG_IMAGE_PLATFORM} -t $IMAGE_NAME -f ${DBLD_DIR}/$DOCKERFILE_PATH ${DBLD_DIR}
+docker build ${DOCKER_BUILD_ARGS} --build-arg=COMMIT=$(git rev-parse --short HEAD) --platform=linux/amd64 --build-arg=ARG_IMAGE_PLATFORM=${ARG_IMAGE_PLATFORM} -t $IMAGE_NAME -f ${DBLD_DIR}/$DOCKERFILE_PATH ${DBLD_DIR}
```

Comment:
this build command was only used on dockerhub and we now build our stuff using github, so I guess the dockerhub based build environment could even be removed.

Can you elaborate why this is needed?

Reply:
also my docker build tool does not have --platform. I am probably using an old version.

The question again arises, where are we using the hooks/build script